content/en/docs/tasks/administer-cluster/manage-resources/quota-memory-cpu-namespace.md

```diff
@@ -141,6 +141,17 @@ Error from server (Forbidden): error when creating "examples/admin/resource/quot
 pods "quota-mem-cpu-demo-2" is forbidden: exceeded quota: mem-cpu-demo,
 requested: requests.memory=700Mi,used: requests.memory=600Mi, limited: requests.memory=1Gi
 ```
+{{< note >}}
+This is an exception for enforcing resource quota on ephemeral-storage.
+
+To make the resource quota work on ephemeral-storage, two things need to be done.
+
+1.admin set the ephemeral-storage resource quota limit
+2.User needs to mention the ephemeral-storage resource quota limit in pod spec.
+
+If the user doesn't specify the ephemeral-storage resource limit in pod spec, then the resource quota is not enforced on ephemeral-storage. Thus the user can use n number of ephemeral storage.
```

Comment:
```suggestion
If the user doesn't specify the ephemeral-storage resource limit in the Pod spec,
the resource quota is not enforced on ephemeral-storage. Thus the user can use
any amount of ephemeral-storage.
```

Reply:
@tengqm i have addressed the suggestions . PTAL